internal/consts/consts.go

```diff
@@ -65,4 +65,11 @@ const (
 	SubscriptionDbusObjectPath = "/com/canonical/UbuntuAdvantage/Manager"
 	// SubscriptionDbusInterface is the interface we are using for access dbus properties.
 	SubscriptionDbusInterface = "com.canonical.UbuntuAdvantage.Manager"
+
+	// Mount related properties.
+
+	// DefaultMountsFilePath is the path in which the mount entries for the user will be / %s should be formatted with the full username.
+	DefaultMountsFilePath string = "/run/adsys/users/%s/mounts"
+	//DefaultUserMountServicePath is the path in which the mount service for the user will be.
```

Comment:
This is going to be removed, but another example on why rereading is really important: you don’t have any space between `//` and the start of the comment, where all the other have. Consistency is important :)